app/views/content_items/briefing.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
```

Comment:
I don't think we want to emphasis the description, so we can just prepend it to the govspeak body and display it below.